docs/horizontal_pod_autoscaling.md

```diff
@@ -0,0 +1,85 @@
+# Horizontal Pod Autoscaling
+
+With Horizontal Pod Autoscaling, Kubernetes automatically scales the number of
+pods in a replication controller, deployment or replica set based on observed
+CPU utilization (or, with alpha support, on some other, application-provided
+metrics).
+
+Kops can assist in setting up HPA and recommends Kubernetes `>= 1.7` and Kops
+`>=1.8`. Relevant reading you will need to go through:
+
+* [Configure The Aggregation Layer][1].
```

Comment:
No periods at the end looks nicer IMO.

Reply:
Fixed in https://github.com/kubernetes/kops/pull/3942/commits/7772c448fe66d1cc0d68e6e8e6245179311ffb20.